src/Renci.SshNet.Tests/Classes/SftpClientTest.ListDirectory.cs

```diff
@@ -89,6 +93,30 @@ public void Test_Sftp_ListDirectory_Current()
             }
         }
 
+#if NET6_0_OR_GREATER
+        [TestMethod]
+        [TestCategory("Sftp")]
+        [TestCategory("integration")]
```

Comment:
Is this test unnecessary now that there is an integration tests project?

Reply:
Yes, I plan to rewrite all integration tests in the new approach, so this will be removed.